app/views/collections/_show_document_list_menu.html.erb

```diff
@@ -1,7 +1,8 @@
 <div class="btn-group">
-  <button class="btn btn-default dropdown-toggle" data-toggle="dropdown">Select an action <span class="caret"></span>
+  <button class="btn btn-default dropdown-toggle" data-toggle="dropdown">Select an action<span class="caret"></span>
   </button>
   <ul class="dropdown-menu dropdown-menu-right">
+    <% if current_user && document.edit_people.include?(current_user.email) %>
```

Comment:
The way we check for permissions is by using the ability class.  So `can? :edit, document`. This way it will check the edit groups as well as the edit users.